tests/main_test.py

```diff
@@ -193,6 +193,52 @@ def test_basic():
     )
 
 
+def test_except():
+
+    # get all columns except one from a CSV
+    eq_test_nrows(
+        "SELECT * EXCEPT (b) FROM csv",
+        [{"a": 1, "c": 3}, {"a": 4, "c": 6}, {"a": 7, "c": 9}],
+        data="a,b,c\n1,2,3\n4,5,6\n7,8,9",
+    )
+
+    eq_test_nrows(
+        "SELECT * EXCEPT(b) FROM csv",
+        [{"a": 1, "c": 3}, {"a": 4, "c": 6}, {"a": 7, "c": 9}],
+        data="a,b,c\n1,2,3\n4,5,6\n7,8,9",
+    )
+
+    # get all columns except two from a CSV
+    eq_test_nrows(
+        "SELECT * EXCEPT(a,c) FROM csv",
+        [{"b": 2}, {"b": 5}, {"b": 8}],
+        data="a,b,c\n1,2,3\n4,5,6\n7,8,9",
+    )
+
+    eq_test_nrows(
+        "SELECT * EXCEPT ( a , c ) FROM csv",
+        [{"b": 2}, {"b": 5}, {"b": 8}],
+        data="a,b,c\n1,2,3\n4,5,6\n7,8,9",
+    )
+
+    # get one column and all except one from a CSV
+    eq_test_nrows(
+        "SELECT b, * EXCEPT (b) FROM csv",
+        [{"b": 2, "a": 1, "c": 3}, {"b": 5, "a": 4, "c": 6}, {"b": 8, "a": 7, "c": 9}],
+        data="a,b,c\n1,2,3\n4,5,6\n7,8,9",
+    )
+
+    eq_test_nrows(
+        "SELECT b AS banana, * EXCEPT (c) FROM csv",
+        [
+            {"banana": 2, "a": 1, "b": 2},
+            {"banana": 5, "a": 4, "b": 5},
+            {"banana": 8, "a": 7, "b": 8},
+        ],
+        data="a,b,c\n1,2,3\n4,5,6\n7,8,9",
+    )
+
+
```

Comment:
I would also add:
- `"SELECT * EXCEPT (nonexisting_column) FROM csv"` (eventually check for warning)
- `"SELECT * EXCEPT () FROM csv"`
- `"SELECT * EXCEPT ("a column defined with quotes", another_without_quotes) FROM csv"`

Reply:
Do you think we shouldn't enforce the use of parenthesis?
Without parenthesis, we are unable to execute queries that include extra columns after the *EXCEPT, e.g.
`"SELECT * EXCEPT (a), a FROM csv"`
I think that enforcing the use of parenthesis isn't such a bad thing.

Reply:
I think we can enforce parenthesis. But I guess that a common mistake will be to skip them when writing the query, especially if it's only 1 column in the except. So, I vote on enforcing parenthesis but check if spyql's output is adequate when users forget to add them.